README.md

```diff
@@ -1,3 +1,5 @@
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/dexidp/dex)
```

Comment:
Can you please move it down below the rest of the badges? Please also add `?style=flat-square`. Thanks!